onnx/defs/schema.h

```diff
@@ -974,6 +978,11 @@ class OpSchemaRegistry final : public ISchemaRegistry {
               << schema.file() << " line " << schema.line() << std::endl;
           fail_schema(err.str());
         }
+        // If opset_version_to_load != 0, only keep the latest one before specified version
```

Comment:
change comment to: Return early if schema for the targeted opset version has already been loaded.